src/plugins/workspace/public/components/add_collaborators_modal/add_collaborators_modal.tsx

```diff
@@ -84,6 +84,26 @@ export const AddCollaboratorsModal = ({
   const [isAdding, setIsAdding] = useState(false);
 
   const handleAddCollaborators = async () => {
+    const singleStarIds = validInnerCollaborators.flatMap(({ id, collaboratorId }) =>
+      collaboratorId.trim() === '*' ? id : []
+    );
+    if (singleStarIds.length > 0) {
+      setErrors(
+        singleStarIds.reduce(
+          (previousErrors, id) => ({
+            ...previousErrors,
+            [id]: i18n.translate('workspace.addCollaboratorsModal.errors.invalidUserFormat', {
+              defaultMessage: 'Invalid {inputLabel} format',
```

Comment:
Nit: do we need a more specific error message?

Reply:
That error message is from designer side. It was dynamic generated from different collaborator types. @xeniatup Do we need to more specific error message for single star? I think it's not a blocking issue, can be updated in the future version.